Export.FCS.sln

```diff
@@ -0,0 +1,27 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Sure. It's not really related to the PR, it's just a quick hack to export metadata using FCS, so it should be moved out.